reference/resources/user.yaml

```
@@ -66,3 +66,9 @@ properties:
     description: The roles this user has been assigned.
     items:
       $ref: "../openapi.yaml#/components/schemas/UserRole"
+
+  merchant_accounts:
+    type: array
+    description: A list of merchant account IDs that the user is assigned to.
```

```suggestion
    description: A list of merchant accounts that the user is assigned to.
```

This is a list of full merchant accounts, right? Not just their IDs?